qml/main.qml

```diff
@@ -40,6 +40,143 @@ ApplicationWindow {
+            name: QT_TR_NOOP("Iisalmi")
+            identifier: "iisalmi"
+            apiName: "waltti"
+            boundarycirclelat: 63.557
+            boundarycirclelon: 27.190
+        }
+        ListElement {
+            name: QT_TR_NOOP("Joensuu region (JOJO)")
+            identifier: "joensuu"
+            apiName: "waltti"
+            boundarycirclelat: 62.601
+            boundarycirclelon: 29.762
+        }
+        ListElement {
+            name: QT_TR_NOOP("Jyväskylä region (Linkki)")
+            identifier: "jyvaskyla"
+            apiName: "waltti"
+            boundarycirclelat: 62.243
+            boundarycirclelon: 25.747
+        }
+        ListElement {
+            name: QT_TR_NOOP("Kajaani region")
+            identifier: "kajaani"
+            apiName: "waltti"
+            boundarycirclelat: 64.227
+            boundarycirclelon: 27.729
+        }
+        ListElement {
+            name: QT_TR_NOOP("Kotka region")
+            identifier: "kotka"
+            apiName: "waltti"
+            boundarycirclelat: 60.461
+            boundarycirclelon: 26.939
+        }
+        ListElement {
+            name: QT_TR_NOOP("Kouvola")
+            identifier: "kouvola"
+            apiName: "waltti"
+            boundarycirclelat: 60.869
+            boundarycirclelon: 26.700
+        }
+        ListElement {
+            name: QT_TR_NOOP("Kuopio region (Vilkku)")
+            identifier: "kuopio"
+            apiName: "waltti"
+            boundarycirclelat: 62.892
+            boundarycirclelon: 27.678
+        }
+        ListElement {
+            name: QT_TR_NOOP("Lahti region (LSL)")
+            identifier: "lahti"
+            apiName: "waltti"
+            boundarycirclelat: 60.984
+            boundarycirclelon: 25.656
+        }
+        ListElement {
+            name: QT_TR_NOOP("Lappeenranta")
+            identifier: "lappeenranta"
+            apiName: "waltti"
+            boundarycirclelat: 61.056
+            boundarycirclelon: 28.185
+        }
+        ListElement {
+            name: QT_TR_NOOP("Mikkeli")
+            identifier: "mikkeli"
+            apiName: "waltti"
+            boundarycirclelat: 61.688
+            boundarycirclelon: 27.274
+        }
+        ListElement {
+            name: QT_TR_NOOP("Oulu region")
+            identifier: "oulu"
+            apiName: "waltti"
+            boundarycirclelat: 65.012
+            boundarycirclelon: 25.471
+        }
+        ListElement {
+            name: QT_TR_NOOP("Rovaniemi (Linkkari)")
+            identifier: "rovaniemi"
+            apiName: "waltti"
+            boundarycirclelat: 66.500
+            boundarycirclelon: 25.714
+        }
+        ListElement {
+            name: QT_TR_NOOP("Vaasa")
+            identifier: "vaasa"
+            apiName: "waltti"
+            boundarycirclelat: 63.096
+            boundarycirclelon: 21.616
+        }
+
+        function getRegion() {
+            var apiName = Storage.getSetting('api');
```

Comment:
Please prevent polling of vehicles on maps in regions that don't have live locations to prevent this: `[W] unknown:77 - file:///usr/share/harbour-jollaopas/qml/js/sirilive.js:77: TypeError: Cannot read property 'URL' of undefined`